tests/cases/compiler/classImplementsPrimitive.ts

```diff
@@ -0,0 +1,17 @@
+// classes cannot implement primitives
+
+class C implements number { }
+class C2 implements string { }
+class C3 implements boolean { }
+class C4 implements Void  { }

```

Comment:
no need for this test; there's no type named Void.

Reply:
Thanks for catching that.